packages/server/src/main/java/software/uncharted/terarium/hmiserver/service/tasks/AMRToMMTResponseHandler.java

```diff
@@ -1,39 +1,18 @@
 package software.uncharted.terarium.hmiserver.service.tasks;
 
-import com.fasterxml.jackson.databind.ObjectMapper;
-import lombok.Data;
+import org.springframework.stereotype.Component;
+
 import lombok.RequiredArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
-import org.springframework.stereotype.Component;
-import software.uncharted.terarium.hmiserver.models.task.TaskResponse;
 
 @Component
 @RequiredArgsConstructor
 @Slf4j
 public class AMRToMMTResponseHandler extends TaskResponseHandler {
 	public static final String NAME = "mira_task:amr_to_mmt";
 
-	private final ObjectMapper objectMapper;
-
 	@Override
 	public String getName() {
 		return NAME;
 	}
-
```

Comment:
We don't actually run any of this code, so we don't need to override anything here

Reply:
Why are we not running it? How do we return the MMT to the front-end?

Reply:
We run it synchronously, it gets the output directly and returns it to the frontend, we aren't persisting it or anything.

Reply:
You mean via `Notifications`?

Reply:
We are persisting those. I mean we don't take the result and write it to a model or document the similar way we do for the other endpoints. It's not a stateful operation, you give input and get output, and it caches.